### PR TITLE
Misc. TF-M+54L support

### PR DESCRIPTION
**Pull request by SebastianBoe**
Misc. TF-M+54L support

### CODE REVIEW

File: platform/ext/target/nordic_nrf/common/nrf54l15/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    target_sources(platform_bl2
+        PRIVATE
+            ${HAL_NORDIC_PATH}/nrfx/mdk/system_nrf54l.c
+    )
+
+    target_compile_definitions(platform_bl2
+        PUBLIC
+            NRF_SKIP_FICR_NS_COPY_TO_RAM
+    )
+endif()
```
Comment (mswarowsky) [inline platform/ext/target/nordic_nrf/common/nrf54l15/CMakeLists.txt:47]: Didn't we remove the bl2 targets?

File: platform/ext/target/nordic_nrf/common/nrf54l15/config.cmake [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+# SPDX-License-Identifier: BSD-3-Clause
+#
+#-------------------------------------------------------------------------------
+
+include(${PLATFORM_PATH}/common/core/config.cmake)
+
+set(SECURE_UART1                        ON         CACHE BOOL      "Enable secure UART1")
+set(NRF_NS_STORAGE                      OFF        CACHE BOOL      "Enable non-secure storage partition")
+set(BL2                                 ON         CACHE BOOL      "Whether to build BL2")
+set(NRF_NS_SECONDARY                    ${BL2}     CACHE BOOL      "Enable non-secure secondary partition")
```
Comment (mswarowsky) [inline platform/ext/target/nordic_nrf/common/nrf54l15/config.cmake:14]: Same here

**Review by mswarowsky — APPROVED** [review]
Only looked at noups commits, 
One question, that can be fixed afterwards and assuming the noup commits get squashed together

File: platform/ext/target/nordic_nrf/common/nrf54l15/CMakeLists.txt (hunk 1 above)
Comment (SebastianBoe) [reply platform/ext/target/nordic_nrf/common/nrf54l15/CMakeLists.txt:47]: bad rebase, will fix

File: platform/ext/target/nordic_nrf/common/nrf54l15/config.cmake (hunk 2 above)
Comment (SebastianBoe) [reply platform/ext/target/nordic_nrf/common/nrf54l15/config.cmake:14]: bad rebase, will fix

**Review by vili-nordic — APPROVED** [review]

**Review by frkv — APPROVED** [review]
Only minor nits on copyright (and a possible superfluous include)

**Review by frkv — APPROVED** [review]
Only minor nits on copyright and possible superfluous include

File: platform/ext/target/nordic_nrf/common/core/target_cfg.c [hunk 3]
```diff
@@ -24,6 +24,8 @@
 #include "region.h"
 #include "array.h"
 
+#include <autoconf.h>
```
[the remark this answers is not in the document]
Comment (SebastianBoe) [reply platform/ext/target/nordic_nrf/common/core/target_cfg.c:27]: noted

**Pull request merged**